core/arch/instr.h

```diff
@@ -989,7 +989,7 @@ instr_set_target(instr_t *cti_instr, opnd_t target);
 INSTR_INLINE /* hot internally */
 #endif
     DR_API
-    /** Returns true iff \p instr's operands are up to date. */
+    /** Returns true if \p instr's operands are up to date. */
```

Comment:
"iff" means "if and only if" -- I think we want to leave it

Reply:
never seen!! ok